### DISCUSSION TITLE
New space creation is cumbersome

### DISCUSSION THREAD

**Discussion by thibaultamartin** [element-hq/element-meta#776] [Product Feedback]
When creating a brand new Space:

- If the Space name is long, text is not warped
- The UI looks very "dry". The "General" and "Random" pre-filled room names get in the way, they should probably only be placeholders
- When I fill only two room names, the third one displays "Room name" with the same style as the two other. It looks like it's going to create a room called "Room name"

<img width="1392" alt="space-create-rooms" src="https://user-images.githubusercontent.com/1688309/196400796-10bc5683-3023-4cdb-a7d1-847dc0c2fcbf.png">

After I created the rooms, I'm brought to the following landing page

<img width="1392" alt="space-welcome" src="https://user-images.githubusercontent.com/1688309/196401066-26a27c57-2836-4ab5-a26b-941e15a16008.png">

The page doesn't do much and looks quite unpolished too. It's never possible to get back to this page.
If I click on the Space avatar in the Space bar, I'm brought to the Space Index which is much more useful:

<img width="1392" alt="space-home-view" src="https://user-images.githubusercontent.com/1688309/196401028-b0a18ee5-0da1-445f-8674-781488f5d5f9.png">

It would probably make sense to remove the landing page and to replace it with the Space index directly.

**Comment by daniellekirkwood** [comment]
References: 


<img width="1392" alt="Screenshot 2022-11-15 at 11 06 08" src="https://user-images.githubusercontent.com/89144281/201896747-55f73e60-9ba5-4bf5-9e8d-59ab2f5636c0.png">

https://user-images.githubusercontent.com/89144281/201896727-60c934a0-d098-4c5d-955d-13c47208bc96.mov
